infraohjelmointi_api/migrations/0046_coordinatornote.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.1.3 on 2023-09-28 07:21
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('infraohjelmointi_api', '0045_auto_project_finances_to_frame_finances'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CoordinatorNote',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('coordinatorNote', models.TextField(blank=True, default='')),
+                ('year', models.CharField(max_length=4)),
+                ('planningClass', models.CharField(max_length=150)),
+                ('updatedByFirstName', models.CharField(max_length=50)),
+                ('updatedByLastName', models.CharField(max_length=50)),
+                ('createdDate', models.DateTimeField(auto_now_add=True)),
+                ('planningClassId', models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, to='infraohjelmointi_api.projectclass')),
+                ('updatedById', models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, to=settings.AUTH_USER_MODEL)),
```

Comment:
For the fields planningClassId and updatedById, can they be renamed by removing the `Id` part. This will follow the theme of the rest of the models. Also improves code readability when accessing other fields through the relation.
Example:
`CoordinatorNoteInstance.updatedById.name` vs `CoordinatorNoteInstance.updatedBy.name`